nx-libs.spec

```diff
@@ -19,6 +19,7 @@ BuildRequires:  gcc-c++
 BuildRequires:  libjpeg-devel
 BuildRequires:  pkgconfig
 BuildRequires:  imake
+BuildRequires:  quilt
```

Comment:
Adding quilt as BR for all RPM-based packaging approaches will not work. RHEL6 and RHEL7 do not have it, for example. Similar issue with openSUSE Leap 42.2

So this patch should come with some if guards, e.g. if 0%{?fedora_version} ... %endif. What do you think?